src/components/slider/slider.ts

```diff
@@ -315,6 +345,34 @@ export class MdSlider implements AfterContentInit {
   clamp(value: number, min = 0, max = 1) {
     return Math.max(min, Math.min(value, max));
   }
+
+  /**
+   * Implemented as part of ControlValueAccessor.
+   * TODO: internal
+   */
+  writeValue(value: any) {
+    this.value = value;
+
+    if (this._sliderDimensions) {
+      this.snapThumbToValue();
+    }
+  }
+
+  /**
+   * Implemented as part of ControlValueAccessor.
+   * TODO: internal
+   */
+  registerOnChange(fn: (value: any) => void) {
+    this._controlValueAccessorChangeFn = fn;
+  }
+
+  /**
+   * Implemented as part of ControlValueAccessor.
+   * TODO: internal
+   */
+  registerOnTouched(fn: any) {
+    this.onTouched = fn;
```

Comment:
Where are you calling this?   It doesn't look like the onBlur function calls it.  Have you checked that the control is correctly set as touched?